src/slices/filtersSlice.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const filtersSlice = createSlice({
+  name: 'filters',
+  initialState: { skills: [], projects: [] },
+  reducers: {
+    setSkillsFilter(state, action) {
+      return { ...state, skills: [ ...action.payload ] };
+    },
+    resetSkillsFilter(state) {
+      return { ...state, skills: [] };
+    },
+    setProjectsFilter(state, action) {
+      return { ...state, projects: [ ...action.payload ] };
```

Comment:
Here you do not need to spread the array. It is only needed when you need to add on the rest, or create a new array reference

```suggestion
      return { ...state, projects: action.payload };
```

The same above in `setSkillsFilter`